docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-fractribution-data-model/index.md

```diff
@@ -28,6 +28,11 @@ The Fractribution package is dependent on the `snowplow_web_page_views` model cr
 ​
 The Snowplow Fractribution dbt package is a tool to perform Attribution Modeling. The name itself comes from `Fractional attribution`, which allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as ROAS (Return On Ad Spend - the amount of revenue that is earned for every dollar spent on advertising) once you have cost data for each marketing channel.
 ​
+:::tip
+
+While the throughout this package we use the term _channel_, it is entirely possible to define this in terms of something else e.g. a campaign.
```

Comment:
I'd add that they can configure the `channel_spend` and `channel_classification` macro to do so.